### DISCUSSION TITLE
OpenFGA Community Meetings

### DISCUSSION THREAD

**Discussion by rhamzeh** [openfga/community#128] [Announcements]
Hey folks!

We're having a monthly community meeting to allow a more direct interaction with the community and allow everyone to get direct access to the team in a more synchronous manner.

* Date: **2nd Thursday of every month**
* Time: **[11am Eastern Time (US)](https://www.worldtimebuddy.com/?qm=1&lid=12,100,5,6,8&h=5&sln=11-12&hf=1) (8AM PT/3PM UTC)**

You can find a link to the calendar [webview](https://calendar.google.com/calendar/embed?src=38a445a2801a268987d9e566b2b8f60d90bb615c4a9678c7c2c992020a166bf2%40group.calendar.google.com&ctz=America%2FMontevideo) and [ical file to import](https://calendar.google.com/calendar/ical/38a445a2801a268987d9e566b2b8f60d90bb615c4a9678c7c2c992020a166bf2%40group.calendar.google.com/public/basic.ics)

[Agenda & Meeting Notes](https://docs.google.com/document/d/1Y6rbD0xpGLVl-7CmeMgxi56_a0ibIQ_RojvWBbT9MZk/edit#) - [Recordings of Previous Meetings](https://www.youtube.com/playlist?list=PLUR5l-oTFZqUneyHz-h4WzaJssgxBXdxB)

We are looking forward to you all joining us!

**Comment by velmohan** [comment]
Has this meeting moved to @3PM UTC?

**Reply by rhamzeh** [reply]
Yep - due to DST changes. It's always 11AM ET, updated the times above.

Thanks for the heads up!

**Reply by velmohan** [reply]
Thank you for confirming @rhamzeh.
